lib/params/ParamsMgr.cpp

```diff
@@ -308,6 +309,17 @@ void ParamsMgr::AddDataMgr(string dataSetName, DataMgr *dataMgr) {
 		addDataMgrNew();
 	}
 
+	ParamsSeparator windowsSep(_rootSeparator, _windowsTag);
```

Comment:
This code is not super clear as to what it accomplishes. A comment may be useful. Especially since it seems to be doing things by creating instances of classes which are then discarded which is not standard. (I am assuming the constructors modify the inputs?)

Reply:
Agreed. I've added a comment. Ultimately it would be nice to refactor the ParamsBase classes to be more intuitive...